### ISSUE TITLE
Videos buffering 

### ISSUE THREAD

**Issue by vehosah** [TeamNewPipe/NewPipe#10525]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.2

### Steps to reproduce the bug

1.go to open newpipe
2. Open video
3. Play the video 

### Expected behavior

I expect the video to play normally 

### Actual behavior

The video is keep buffering every two seconds, no matter the connection, not matter if it is 720 or 480, not matter how much I clean the cache 

### Affected Android/Custom ROM version

Android 12

### Affected device model

Motorola g32 /Nokia 5.4 

### Additional information

I also tried the version 0.26.0 nah nothing works

**Comment by vehosah** [comment]
Sorry I don't have another device to try

**Comment by vehosah** [comment]
I don't know any friend who uses newpipe, so I can't ask a friend

**Comment by vehosah** [comment]
Oh well I asked a friend to try it, it's the same buffering hell as me  :(

**Comment by vehosah** [comment]
On their own

**Comment by vehosah** [comment]
On a Samsung galaxy a14 on Android 12

**Comment by vehosah** [comment]
Wifi,both of us

**Comment by vehosah** [comment]
Wifi, both of us

**Comment by vehosah** [comment]
Greece

**Comment by vehosah** [comment]
> Alright, let's reopen this and see if others have a similar experience.
> 
> What's your region?

Greece

**Comment by Feuerswut** [comment]
Try vpn-ing to Germany? It works fine for me?
Is there some specific legislation in Greece that forces YouTube to implement region-specific features?...

In Germany, there is NetzDG for example, but this does not affect NP.

**Comment by vehosah** [comment]
> Try vpn-ing to Germany? It works fine for me. Is there some specific legislation in Greece that forces YouTube to implement region-specific features?...
> 
> In Germany, there is NetzDG for example, but this does not affect NP.

I tried VPN to Germany, I don't see any difference

**Comment by SassyTheSamsquatch** [comment]
Smae issue here

**Comment by shreyas-a-s** [comment]
Can you try changing ExoPlayer Playback load interval size to 1KiB? Seems to make the loading times a bit better for me.

**Comment by vehosah** [comment]
> Can you try changing ExoPlayer Playback load interval size to 1KiB? Seems to make the loading times a bit better for me.

A little bit, not that it fixed it actually

**Comment by vehosah** [comment]
Yeah I would like to remind that the problem became ten times worse

**Comment by vehosah** [comment]
Yeah I tried it it's the same
